Makefile

```diff
@@ -65,17 +66,37 @@ lint: ## Lint go source code
 .PHONY: lint
 
 # --- frontend -----------------------------------------------------------------
-frontend: | $(O) ## Build frontend, typically iterate with npm and inside frontend
+frontend: tiny wasm-exe | $(O) ## Build frontend, typically iterate with npm and inside frontend
 	rm -rf $(O)/public
 	cp -r frontend $(O)/public
+	rm -f $(O)/public/evy.wasm
+	rm -f $(O)/public/wasm_exec.js
+	cp $(O)/evy.wasm $(O)/public/
+	cp $(O)/wasm_exec.js $(O)/public/
```

Comment:
^ this is all pretty horrible, but i didn't find a better way of making firebase play along. Interestingly it all worked locally, just on CI / with the firebase github action I had issues. It must be something todo with following symlinks.

Initially I had:
```
frontend/evy.wasm -> ../out/evy.wasm
frontend/wasm_exec.js -> ../out/wasm_exec.js
```
I copied (and tried many flags) the files to `out/public` on `make frontend`.
and I had `firebase/public -> ../out/public`...